app/src/main/res/values/styles.xml

```diff
@@ -14,4 +14,10 @@
     <style name="CustomBottomSheetStyle" parent="Widget.Design.BottomSheet.Modal">
         <item name="android:background">@drawable/rounded_dialog</item>
     </style>
+
+    <style name="Widget.BottomNavigationView"
```

Comment:
Custom widgets should follow the `Widget.[projectName].BottomNavigationView` naming convention. In this case: `Widget.AdventureMaps.BottomNavigationView`